js-core/lesson1.md

```diff
@@ -1,19 +1,176 @@
 # JavaScript Core 1
-** What we will learn today?**
+
+![Status: Draft](https://camo.githubusercontent.com/997591db1749880b8b23c96b8f788e69af09c04d/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f7374617475732d64726166742d6461726b7265642e737667)
+
+**What we will learn today?**
+
 - Getting started with JavaScript
+- Simple data types and expressions
 - Variables, assignment and comparison
 - Conditionals
   - if/else, switch and ternary operator
 - Arrays and loops
 - Objects
+
 ---
 
-## Topic 1
-High level explanation of Topic 1 - rely on the exercises
-> **Exercise**: Always have exercises
+## Getting started with JavaScript
+
+We will spend most of this lesson in Codepen. To get started, go to [codepen.io/pen/](codepen.io/pen/) to create a new Pen. On the bottom left, click the "Console" button to open the console.
+
+![Codepen console](assets/codepen-console.png)
+
+You can place your cursor right behind the `>` sign. Type a simple expression, such as `2 + 2` and hit enter. You will see expression, as well as its result, in the window above.
+
+![Entering expressions in the Codepen console](assets/repl.gif)
+
+You will be able to follow most of this session along just entering expressions in the console like this.
+
+## Simple data types & Expressions
+
+### Numbers
+
+Let’s start with something seemingly simple - numbers. Here are some:
+
+```js
+1
+15
+3195803798
+1.4
+0.0000005
+0
+-23
+```
+
+You will see that there are whole numbers (no decimal point) and real numbers (decimal point), which in JavaScript are so-called “floating point” numbers. Numbers can be positive or negative and they support all the basic math operators that you would expect:
+
+```js
+2 + 2 // 4
+2 - 5 // -3
+2 * 3 // 6
+10 / 2 // 5
+3 * -2 // -6
+```
+
+Operators have the same precedence as in algebra: `*` and `/` have higher precedence than `+` and `-`.
+
+- **TODO:** Do math exercise
+- **TODO:** What about Infinity, Math.PI, IEEE floating point bullshit, NaN
+
+### Strings
+
+Strings represent any sort of text. They are delimited by single quotes (`'`) or double quotes (`”`) and can be of any length.
```

Comment:
I wasn't sure either. @kabaros what is our approach to ES6+ things like template strings? This also informs if we introduce `const` and `let` early on, which really goes down a rabbit whole of "scope" wayyy to early, I think.